scripts/run.sh

```diff
@@ -64,7 +64,10 @@ echo "Test pod exited with code:${podExitCode}"
 
 #Collect artifacts
 retrive-artifacts
+retrieveArtifactsExitCode=$?
+
 
 #Clean up
 delete_lease
-exit ${podExitCode}
+exitCode=$(${podExitCode}+${retrieveArtifactsExitCode})
```

Comment:
By adding the two exit codes together we create logic that allows us to
check multiple error states at once. So if one exit code is nonzero and the
other is zero then the script will exit with a nonzero code
On Jun 13, 2016 5:52 PM, "Matthew Fisher" <notifications@github.com> wrote:

> In scripts/run.sh
> <https://github.com/deis/e2e-runner/pull/17#discussion_r66889091>:
>
> >
> >  #Clean up
> >  delete_lease
> > -exit ${podExitCode}
> > +exitCode=$(${podExitCode}+${retrieveArtifactsExitCode})
>
> what does this do? I don't think I've ever seen an addition operator used
> in bash
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/deis/e2e-runner/pull/17/files/12f127c971138b73947da357c3b79cb33f507b69#r66889091>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAaRGNACHkM-txLdguG1zKEmlqa1aVw-ks5qLe01gaJpZM4I00B7>
> .
>